### PR TITLE
feat: Add script for pre-labeling datasets

### PR DESCRIPTION
**Pull request by imenelydiaker**

### CODE REVIEW

File: scripts/pre_labeling_dataset.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    "pineapple": ["ovaloid", "brown", "yellow"],
+    "bell pepper": ["green", "red", "yellow", "stem"],
+    "raddish": ["red", "stem"],
+    "onion": ["white", "sphere"],
+    "pear": ["green"],
+    "pomegranate": ["red", "seed"],
+    "sweetcorn": ["seed", "yellow"],
+    "sweetpotato": ["orange", "brown", "ovaloid"],
+}
+
```
Comment (Xmaster6y) [inline scripts/pre_labeling_dataset.py:57]: Should be in `constants.py`

File: scripts/pre_labeling_dataset.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        metadata = get_metadata(hf_api, split=split)
+
+        logger.info("Pre-label concepts")
+        df = pd.DataFrame.from_records(metadata)
+
+        assert len(df["class"].unique()) == len(CLASS_CONCEPTS_VALUES.keys())
+
+        for key, values in CLASS_CONCEPTS_VALUES.items():
+            for val in values:
+                df.loc[df["class"] == key, val] = True
```
Comment (Xmaster6y) [inline scripts/pre_labeling_dataset.py:110]: will erase previous labe

Comment (Xmaster6y) [inline scripts/pre_labeling_dataset.py:110]: can be multicolumn

**Review by Xmaster6y — CHANGES_REQUESTED** [review]

File: scripts/pre_labeling_dataset.py (hunk 2 above)
Comment (Xmaster6y) [reply scripts/pre_labeling_dataset.py:110]: only replace none

File: scripts/pre_labeling_dataset.py (hunk 1 above)
Comment (imenelydiaker) [reply scripts/pre_labeling_dataset.py]: Updated ✅

File: scripts/pre_labeling_dataset.py (hunk 2 above)
Comment (imenelydiaker) [reply scripts/pre_labeling_dataset.py]: Updated ✅

Comment (imenelydiaker) [reply scripts/pre_labeling_dataset.py]: Actually it's hard coded, I assume we run this only once, so I ignored the 4 classes that are already labeled

Comment (imenelydiaker) [reply scripts/pre_labeling_dataset.py]: I can make it update only null values

Comment (Xmaster6y) [reply scripts/pre_labeling_dataset.py]: I labelled outside of these classes.

**Review by Xmaster6y — APPROVED** [review]

**Pull request merged**